services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
```

Comment:
Consider importing `datetime` from the standard library instead of `django.utils.datetime_safe`, as the latter is typically used for compatibility with older Python versions. Use `from datetime import datetime` instead.